packages/openapi-react-query/src/index.ts

```diff
@@ -115,9 +115,10 @@ export default function createClient<Paths extends {}, Media extends MediaType =
     const mth = method.toUpperCase() as Uppercase<typeof method>;
     const fn = client[mth] as ClientMethod<Paths, typeof method, Media>;
     const { data, error } = await fn(path, { signal, ...(init as any) }); // TODO: find a way to avoid as any
-    if (error || !data) {
```

Comment:
I think it's best we don't check for data at all. 
If it's undefined - react-query will throw an error for us. Other falsy values are fine